config/packages/prod/monolog.yaml

```diff
@@ -8,8 +8,9 @@ monolog:
             buffer_size: 50 # How many messages should be saved? Prevent memory leaks
         nested:
             type: stream
-            path: "%kernel.logs_dir%/%kernel.environment%.log"
+            path: "%kernel.logs_dir%/%kernel.environment%.log" # php://stderr
```

Comment:
This is different to the new symfony default as I thought the symfony default doesn't make sense for us, as most of our users have more traditional setup then a cloud setup:  See https://github.com/symfony/recipes/pull/894

Reply:
👍